src/base.py

```diff
@@ -29,13 +32,26 @@ def change_to_base(n: int, b: int) -> str:
     """
     assert 2 <= b <= 16
     lst = []
+    ## assert 0/negative values
+    is_negative = 0
+    if n == 0:
+        return "0"
+    elif n < 0 : 
+        n = -1*n
+        is_negative = 1
```

Comment:
We have boolean values in Python, so there is no need to use 0 and 1. This could just be `is_negative = True` and then you initialised it with `False` instead of 0.

I think that I would use a string instead, though. The only place where you need `is_negative` is where you set the sign in front of the result. So you could do

```python
sign = ""
if n < 0:
    sign = "-"
```

and later, when you need a sign, you use `sign + "".join(...)`.